pipe/section/section_impls/typecast_transformer/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "typecast_transformer"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+section = { path = "../../" }
+arrow = { version = "49", features = ["prettyprint"] }
```

Comment:
Feels like we need to re-export arrow through arrow_msg, WDYT?

Reply:
I think that would help with keeping the versions consistent.